src/examples/image_carousel/imageViewer.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Carousel } from 'react-responsive-carousel';
+
+const URLREGEX = new RegExp("((http|https)(:\/\/))?([a-zA-Z0-9]+[.]{1}){2}[a-zA-Z0-9]+(\/{1}[a-zA-Z0-9]+)*\/?", "igm");
+const B64REGEX = new RegExp("^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)?$", "igm");
```

Comment:
My regex fu is not great but this is what I found on MSDN`/^(?:[A-Za-z\d+\/]{4})*?(?:[A-Za-z\d+\/]{2}(?:==)?|[A-Za-z\d+\/]{3}=?)?$/`
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob

A comment here explaining where the regexs came from would be useful for future travelers.

Reply:
Updated the regex and left a comment about where it came from